app/javascript/dashboard/routes/dashboard/settings/inbox/components/InputRadioGroup.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <label class="radio-group-label">{{ label }}</label>
+    <div class="radio-group">
+      <div v-for="item in items" :key="item.id">
+        <input
+          name="radio-input"
+          type="radio"
+          :checked="item.checked"
+          @change="action(item)"
```

Comment:
@nithindavid should we be using the `v-model` way to handle data? or would this suffice?

Reply:
Not sure if it would work since we are working with an item in an array. We could try @aswindevps